flight/Modules/OnScreenDisplay/onscreendisplay.c

```diff
@@ -1276,6 +1263,8 @@ void render_user_page(OnScreenDisplayPageSettingsData * page)
 		AccelsData accelsData;
 		AccelsGet(&accelsData);
 		// apply low pass filter to reduce noise bias
+		static AccelsData accelsDataAcc = {};
```

Comment:
It indeed is a gcc-ism.  But imo preferable to coercing a 0 in there or increasing scope and using bzero.